### PR TITLE
【Hackathon 5th No.45】映射文档73, 74

### PR DESCRIPTION
**Pull request by longranger2**
73 torch.optim.Optimizer.zero_grad，pytorch参数更多，标注到 pytorch_api_mapping_cn.md 文件中。
74 torch.nn.modules.batchnorm._NormBase，功能缺失，标注到 pytorch_api_mapping_cn.md 文件中。

- https://github.com/PaddlePaddle/Paddle/issues/57262

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/optimizer/torch.optim.Optimizer.zero_grad.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+### [torch.optim.Optimizer.zero_grad](https://pytorch.org/docs/stable/generated/torch.nn.Module.html#torch.optim.Optimizer.zero_grad)
+
+```python
+torch.optim.Optimizer.zero_grad(set_to_none=True)
+```
+
+### [paddle.optimizer.Optimizer.clear_grad](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/optimizer/Optimizer_cn.html#clear-grad)
+
+```python
+paddle.optimizer.Optimizer.clear_grad()
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/optimizer/torch.optim.Optimizer.zero_grad.md:12]: 这个还有一个参数：
infoflow 2024-01-22 11-58-24

File: docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md [hunk 2]
```diff
@@ -524,6 +524,7 @@
 | REFERENCE-MAPPING-ITEM(`torch.nn.parameter.Parameter`, https://github.com/PaddlePaddle/docs/tree/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/utils/torch.nn.parameter.Parameter.md) |
 | NOT-IMPLEMENTED-ITEM(`torch.nn.MultiLabelMarginLoss`, https://pytorch.org/docs/stable/generated/torch.nn.MultiLabelMarginLoss) |
 | NOT-IMPLEMENTED-ITEM(`torch.nn.CTCLoss`, https://pytorch.org/docs/stable/generated/torch.nn.CTCLoss.html#torch.nn.CTCLoss) |
+| [torch.nn.modules.batchnorm._NormBase]([https://pytorch.org/docs/stable/generated/torch.nn.MultiLabelMarginLoss](https://github.com/pytorch/pytorch/blob/a1b3b5748f5b8edfda1ff0fb159037eed3b669f1/torch/nn/modules/batchnorm.py#L17)) | 功能缺失 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:527]: 这个你检查下这个文件里的Norm基类class：python/paddle/nn/layer/norm.py
